solutions/security-enforced/version.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_version = ">= 1.3.0"
```

Comment:
I suggest you update to `>= 1.9.0` so we can use the terraform 1.9 cross variable validation feature.
We will need to use it in many input variables in variables.tf. For an idea of how to use it, see my branch (WIP): https://github.com/terraform-ibm-modules/terraform-ibm-scc/blob/DA/solutions/fully-configurable/variables.tf